webcompat/api/uploads.py

```diff
@@ -100,6 +106,10 @@ def save(self):
             save_parameters['save_all'] = True
         # unpacking save_parameters
         self.image_object.save(file_dest, **save_parameters)
+        # Creating the thumbnail
+        size = (150, 150)
+        self.image_object.thumbnail(size, Image.ANTIALIAS)
```

Comment:
Results appear to be basically the same. `NEAREST` is the fastest, but largest (at 4.7% of original size). Second fastest is `BILINEAR`, and smallest (at 3.6% of original size).

I would sort of lean towards speed over filesize and pick `NEAREST` -- but both are reasonably fast and small.